pkg/extensions/search/common/oci_layout.go

```diff
@@ -338,12 +338,12 @@ func (olu BaseOciLayoutUtils) CheckManifestSignature(name string, digest godiges
 }
 
 func (olu BaseOciLayoutUtils) GetImageLastUpdated(imageInfo ispec.Image) time.Time {
```

Comment:
I think this should be a function instead of a method of `olu`. It doesn't need anything from this object, doesn't read from the disk.
We we could call it in `GetImageTagsWithTimestamp` instead of duplicating the code. Or at least implement a new function in common.go and call it in both these existing places if we don't want to be disruptive..